assets/_scss/components/_accordions.scss

```diff
@@ -0,0 +1,46 @@
+#accordion-menu {
+  border-color: $color-grey-600;
+  border-style: solid;
+  border-width: 1px;
+  border-bottom-width: 0;
+  color: $color-grey-900;
+  margin: 0;
+  padding: 0;
+
+  li {
+    background-color: $color-grey-200;
+    border-bottom-width: 1px;
+    font-family: $font-sans;
+    list-style: none;
+    margin: 0;
+
+    &.hidden {
+      .content {
+        display: none;
+      }
+
+      p {
+        background: url('../img/arrow-right.png') no-repeat;
+        background-position: 15px;
+      }
+    }
+
+    p {
```

Comment:
Same comment as above for here. This using a p tag here has the added problem of hurting accessibility because it's not going to be shown as a link or button.